src/include/pixels/RGBPixel.hpp

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+#include "Pixel.hpp"
+
+class RGBPixel : public Pixel
+{
+public:
+	RGBPixel() = delete;
```

Comment:
No need from this. You have other constructors, so the default won't be generated anyway.

Reply:
I think it's fine this way.